android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java

```diff
@@ -708,10 +708,11 @@ public void onGetAvailableCountries(String[] countries) {
 
     @CalledByNative
     public void OnGetAdsAccountStatement(boolean success, double next_payment_date,
-            int ads_received_this_month, double earnings_this_month, double earnings_last_month) {
+            int ads_received_this_month, double min_earnings_this_month,
+            double max_earnings_this_month, double earnings_last_month) {
         for (BraveRewardsObserver observer : mObservers) {
             observer.OnGetAdsAccountStatement(success, next_payment_date, ads_received_this_month,
-                    earnings_this_month, earnings_last_month);
+                    min_earnings_this_month, max_earnings_this_month, earnings_last_month);
```

Comment:
nit : java method param should be in camelcase

Reply:
updated